source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -127,30 +128,6 @@ void BioXASAppController::cleanMoveImmediatelyAction()
 	moveImmediatelyAction_ = 0;
 }
 
-void BioXASAppController::onUserConfigurationLoadedFromDb()
```

Comment:
Hmmm. The BioXAS implementation differs from the default implementation in CLSAppController by handling multiple detectors. Could this slot be virtual? That way different applications can reimplement as necessary.